backend/dal/Security/Permissions.cs

```diff
@@ -64,6 +64,12 @@ public enum Permissions
         ProjectEdit = 32,
 
         [Display(GroupName = "project", Name = "project-delete", Description = "Can delete projects.")]
-        ProjectDelete = 33
+        ProjectDelete = 33,
+
+        [Display(GroupName = "project", Name = "reports-view", Description = "Can view project reports.")]
```

Comment:
You will need to create a new role **SRES Financial Manager**, add these claims to it.  This will require updating the database.  We should be able to run the keycloak-sync tool to get them into keycloak afterwards.